src/core/data/index.ts

```diff
@@ -324,13 +333,15 @@ export interface ContactInfo {
   contactEmail?: string
 }
 
-export interface TaxPayer extends ContactInfo {
+export interface TaxPayer<D = Date> extends ContactInfo {
   filingStatus?: FilingStatus
-  primaryPerson?: PrimaryPerson
-  spouse?: Spouse
-  dependents: Dependent[]
+  primaryPerson?: PrimaryPerson<D>
+  spouse?: Spouse<D>
+  dependents: Dependent<D>[]
 }
 
+export type TaxPayerDateString = TaxPayer<string>
```

Comment:
I think the build error you're seeing shows that you just need to do this same thing for each of the other types you've modified this way.

Reply:
I'm not sure what you mean, I think I've already done that for Person, PrimaryPerson, Spouse, and Dependent.

Reply:
Ah I see. Ok will have to debug  https://github.com/ustaxes/UsTaxes/runs/5882167289?check_suite_focus=true